gotoalert/slack.py

```diff
@@ -127,6 +127,14 @@ def send_event_report(event):
             # Fermi events should have a duration parameter
             extra_details += ['Duration: {}'.format(event.duration.capitalize()),
                               ]
+    elif event.type == 'NU':
+        # NU events provide a given location
+        extra_details = ['Signalness: {:.3f} probability to be astrophysical in origin'.format(event.signalness),
+                         'FAR: {:.3f} per yr'.format(event.far),
```

Comment:
Minor change to match LIGO events.

```suggestion
        extra_details = ['Signalness: {:.0f}% probability to be astrophysical in origin'.format(event.signalness * 100),
                         'FAR: ~1 per {:.1f} yrs'.format(1 / event.far),
```